packages/spruce-skill-server/factories/sequelize.ts

```diff
@@ -175,5 +175,5 @@ export default (
 		}
 	}
 
-	ctx[key] = { models, sequelize, sync }
+	ctx[key] = { models, sequelize, sync, Sequelize }
```

Comment:
It seems like we shouldn't need to set `Sequelize` here since we can just import it elsewhere if we need it

Reply:
Inside of booking, we invoke `ctx.db.sequelize.literal(...` and get a "Calling method on static function".

Booking will have a corresponding PR with, `ctx.db.Sequelize.literal(...`.

Reply:
Yeah, since we're not initializing anything and it's just the static methods I was thinking we'd just import it:

```
import { literal } from 'sequelize'

ctx.db.models.User.findOne({
  where: literal(...)
})
```